addons/skin.estuary/language/resource.language.ar_sa/strings.po

```diff
@@ -27,7 +27,7 @@ msgstr "Estuary هو المظهر الافتراضي ل كودي الإصدار
 
 msgctxt "Addon Disclaimer"
 msgid "Estuary is the default skin for Kodi, removing it may cause issues"
-msgstr ""
+msgstr "‫Estuary هو المظهر الافتراضي ل كودي, إزالته قد تسبب بعض المشاكل"
```

Comment:
Think here was gone something wrong and maybe request to close?

Reply:
This a new warning in Github.
The translation includes a right to left embedding character: `U+202B`.
A lot of our translations include this character.
I tested some of them and didn't find any issues, but I don't know the exact issues it might lead to.

For this string, I removed the character by copying some of the translation from another string (with the exact same words).